olive/passes/onnx/quantization.py

```diff
@@ -13,6 +13,8 @@
 from onnxruntime.quantization.calibrate import CalibrationMethod
 from onnxruntime.quantization.preprocess import quant_pre_process
 
+from neural_compressor import quantization, PostTrainingQuantConfig
```

Comment:
Can we make `neural-compressor` an extra by doing a try import of these at `_run_for_config` line 552? Like in this openvino pass https://github.com/microsoft/Olive/blob/main/olive/passes/openvino/conversion.py#L57  

We could then add neural-compressor to extras here https://github.com/microsoft/Olive/blob/main/setup.py#L33 instead of making it a requirement.

Reply:
Hi, I have moved `from neural_compressor import quantization, PostTrainingQuantConfig` into `_run_for_config`. Please help add neural-compressor to extras, Thanks!

Reply:
Thanks for moving it. Sorry, I was unclear in my comment. Can you please add the extra yourself to setup.py? Maybe you can call the extra as `inc`.

Reply:
Sorry I misunderstood you. I have added the extra as `"inc": ["neural-compressor"],`